### PR TITLE
Add sc5 docker-compose-prometheus-grafana example

### PR DESCRIPTION
**Pull request by amckenzie132**
Add a docker-compose example that runs Sauce Connect 5, Prometheus, and Grafana with the Sauce Connect dashboard.

### CODE REVIEW

File: examples/sauce-connect-5/docker-compose-prometheus-grafana/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+services:
+  sauce-connect:
+    deploy:
+      replicas: 1
+      restart_policy:
+        delay: 30s
+        max_attempts: 5
+        condition: on-failure
+        window: 60s
+    image: "saucelabs/sauce-connect:5.0.0"
```
Comment (alexh-sauce) [inline examples/sauce-connect-5/docker-compose-prometheus-grafana/docker-compose.yaml:17]: I think we can future proof the image name by using `"saucelabs/sauce-connect:5"`

File: examples/sauce-connect-5/docker-compose-prometheus-grafana/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
+    ports:
+      - 9090:9090
+
+  grafana:
+    image: grafana/grafana:10.1.2
+    restart: always
+    user: "472"
```
Comment (alexh-sauce) [inline examples/sauce-connect-5/docker-compose-prometheus-grafana/docker-compose.yaml:47]: Where does the 472 come from?

Comment (amckenzie132) [reply examples/sauce-connect-5/docker-compose-prometheus-grafana/docker-compose.yaml:47]: Debugging attempt, removed it and it works. Ty

File: examples/sauce-connect-5/docker-compose-prometheus-grafana/docker-compose.yaml (hunk 1 above)
Comment (amckenzie132) [reply examples/sauce-connect-5/docker-compose-prometheus-grafana/docker-compose.yaml:17]: Updated to 5.0 until we have 5

**Review by alexh-sauce — APPROVED** [review]
LGTM

**Pull request merged**